src/main.py

```
@@ -0,0 +1,18 @@
+from fastapi import FastAPI
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)